helpers/preview/bin/preview-rst

```diff
@@ -0,0 +1,313 @@
+#!/bin/sh -e
+
+# Licensed to Crate (https://crate.io) under one or more contributor license
+# agreements.  See the NOTICE file distributed with this work for additional
+# information regarding copyright ownership.  Crate licenses this file to you
+# under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+# However, if you have executed another commercial license agreement with Crate
+# these terms will supersede the license and you may use the software solely
+# pursuant to the terms of the relevant commercial agreement.
```

Comment:
Don't know if this is still a thing? I wouldn't mind just leaving it out.

Reply:
we use this in crate/crate. I want to mirror that repo. if we remove it there, I can remove it from our secondary repos (like crate-docs-license, crate-python, and all the other drivers)

Reply:
made a pr for this question: https://github.com/crate/crate-docs/issues/76